doc/nrf/libraries/fem_al/index.rst

```diff
@@ -0,0 +1,48 @@
+.. _fem_al_lib:
+
+FEM Abstraction Layer
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The library is a wrapper for the MPSL FEM.
+
+Overview
+********
+
+The library is designed to simplify low level usage of a FEM hardware.
+
+Requirements
+************
+
+The library requires the usage of a FEM and MPSL functionality.
```

Comment:
This does not sound like a proper requirement, as the lib is meant to be used with FEM and MPSL, which is stated already before. If there are no other requirements, you might as well leave this section out.